tools/modules/framework/module_data_files.sh

```
@@ -0,0 +1,391 @@
+	if [ "$group" != "unknown" ] && [ -n "$id" ]; then
+		conf_file="$tools_dir/dev/tests/${id}.conf"
+	fi
+
+	# Create the parent directory if it doesn't exist
+	mkdir -p "$(dirname "$conf_file")"
+
+	local commands
+	IFS=' ' read -r -a commands <<< "${module_options["$feature,example"]}"
+
+	if [[ $parent == "software" ]]; then
+		if [[ " ${commands[@]} " =~ " help " && " ${commands[@]} " =~ " status " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)